public/stylesheet.css

```diff
@@ -1,11 +1,14 @@
 * {
   box-sizing: border-box;
+  font: inherit;
   margin: 0;
+  outline: 0;
   padding: 0;
+  vertical-align: baseline;
 }
```

Comment:
Meyer's reset includes `border: 0`, why didn't you include it here?

Reply:
I have added it, I  mistakenly omitted it earlier😬